content/docs/00-release-notes.md

```diff
@@ -73,6 +73,11 @@ Spectro Cloud Palette 3.0.0 is released with [Native Edge](/clusters/edge/native
 * Open Policy Agent version 3.9.0
 * AWS EBS  CSI version 1.10.0
 
+**Known Issues**
+
+* [PPD-359](https://spectrocloud.atlassian.net/browse/PPD-359): While deploying multiple applications on a Sandbox cluster, the first installed application gets stuck at deployment, leads to other applications within the same Sandbox cluster getting stuck.
```

Comment:
Customers will not have access to this internal link, therefore we should not expose it as it creates bad UX. Is there a public-facing resource we can point to? Otherwise we best remove it,

Reply:
While deploying multiple applications on a Sandbox cluster, if one of application deployment gets stuck can cause other applications within the same Sandbox cluster getting stuck.